coralproject/talk
# [next] Support for Rich Text

## What does this PR do?
- Integrate https://github.com/coralproject/rte
- Sanetize and display HTML Content
- Make each test in integration suites independent
- Fix some translations and tests
- Improve accessibility 

## How do I test this PR?

- Open embed stream
- See rich text editor
